Target-MVD-Mathias/ViewControllers/SignUpViewController.swift

```diff
@@ -234,11 +226,16 @@ extension SignUpViewController: UITextFieldDelegate {
     } else {
       hideErrorInForm(textField: passwordTextField, errorLabel: passwordErrorLabel)
       
-      if confirmPasswordIsInvalid() && !firstTimeEditingPassword {
-        showErrorInForm(textField: confirmPasswordTextField, errorLabel: confirmPasswordErrorLabel)
-      } else {
-        hideErrorInForm(textField: confirmPasswordTextField, errorLabel: confirmPasswordErrorLabel)
-      }
+      let confirmPasswordError = confirmPasswordIsInvalid() && !firstTimeEditingPassword
+      toggleErrorInForm(error: confirmPasswordError, textField: confirmPasswordTextField, errorLabel: confirmPasswordErrorLabel)
+    }
+  }
+  
+  fileprivate func toggleErrorInForm(error: Bool, textField: UITextField, errorLabel: UILabel) {
+    if error {
```

Comment:
There is not best practice for this operator. Its up to you, actually I don't use it for method calls either.